app/talk/lib/create-board-form.cjsx

```diff
@@ -2,15 +2,57 @@ React = require 'react'
 ROLES = require './roles'
 talkClient = require '../../api/talk'
 
+intersperse = (arr, item) ->
+  arr.reduce((acc, x, i) ->
+    if arr.length - 1 isnt i
+      acc.concat(x, item)
+    else
+      acc.concat(x)
+  , [])
+
+getUserRoleNames =  (user, section) ->
+  talkClient.type('roles').get
+    user_id: user.id,
+    section: ['zooniverse', section],
+    page_size: 100
+  .then (roles) ->
+    roles.map (role) -> role.name
+
+mostPowerfulRole = (roleNames, rolesList) ->
```

Comment:
Only admin and moderator can be accessing this view.  Would it be easier to just check for admin and assume moderator otherwise?